requirements.txt

```diff
@@ -5,10 +5,22 @@
 click
 Sphinx
 coverage
-# awscli
 flake8==5.0.*
 flake8-black==0.3.*
 flake8-import-order==0.18.*
 pre-commit==2.21.*
 isort==5.11.*
 python-dotenv>=0.5.1
+
+# extra requirements
+omegaconf==2.3.0
+pytorch-lightning==1.8.6
+timm==0.6.12
+
+# pytorch requirements
```

Comment:
What does the "+cu117"? do?

Reply:
It loads the pytorch version with cuda that we use to train and evaluate with gpu.

Reply:
This version will most likely work with `google cloud`, but we can change it later if it doesn't